README.md

```diff
@@ -3,20 +3,22 @@
 # CORE
 Core is the backend of MentorPaired, an app which connects software engineering mentors to mentees who seek to learn any programming language.
 
-### Tech
+#### Tech
 Core is written in Python3 and Django 3.0.3.
 
-### Installation
+## Installation
 
 #### Ubuntu 18.04 Users
 
- Please install and set up the following packages first, upgrade if you find the package is already installed:
+ Please install and set up the following packages first. Upgrade if you find the package is already installed:
 
 * Python3. Run the 'python3 -V' command to see the version you have installed.
 
-* [PostgreSQL](https://www.postgresql.org/) (Ensure the server is running).
+* Git
```

Comment:
It's assumed that whoever cloned this project already has Git installed. No need to explicitly add this.